tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/OrderExtensionsTests.cs

```diff
@@ -161,8 +161,8 @@ public static void IsSectionStatusCompleteComplete_NullOrder_ReturnsFalse()
         [CommonInlineAutoData(true, true, true, true, true, true, true)]
         [CommonInlineAutoData(true, true, true, true, false, true, true)]
         [CommonInlineAutoData(true, true, false, false, false, false, false)]
-        [CommonInlineAutoData(true, true, false, true, true, false, false)]
-        [CommonInlineAutoData(true, false, true, false, false, true, false)]
+        [CommonInlineAutoData(true, true, false, true, true, false, true)]
+        [CommonInlineAutoData(true, false, true, false, false, true, true)]
```

Comment:
Based on my comments about the scenarios above, these two test cases are also now incorrect so should be reverted.

Reply:
Revert is done